master/changes.html

```diff
@@ -1007,12 +1007,17 @@ <h3 id="svgdom">SVG Document Object Model (DOM)</h3>
         <li>Add Geometry Interfaces Module Level 1 as a requirement.</li>
       </ul>
     </li>
+    <li>Clarify that SVG DOM requirements extend to elements defined in other specifications.</li>
+    <li>Add a requirement for handling unknown elements in the SVG namespace:
+      "must" handle as browsers currently do (as SVGElement instances);
+      "should" implement SVGUnknownElement interface (though that is at risk).
+    </li>
     <li>Remove the special rule for initializing <a>SVGTextContentElement::textLength</a>,
       in favor of a clearer definition of the attribute's dynamic initial value.
     </li>
     <li>Clarify that rules about initializing objects
       only apply when reflecting <code>(none)</code> initial values,
-      and apply to interfaces no longer defined in this specification.
+      and also apply to interfaces defined in other specifications (e.g., Geometry).
```

Comment:
much better

Reply:
My own bad wording from the previous commit. Read it a second time & that's not quite what I meant to say!